core/src/main/java/hudson/model/RunParameterDefinition.java

```diff
@@ -164,20 +164,25 @@ public ParameterValue getDefaultParameterValue() {
         }
 
         Run<?,?> lastBuild = null;
+        Job project = getProject();
+
+        if (project == null) {
+            return null;
```

Comment:
Probably it's reasonable to add some logging for this case since it's an illegal condition

Reply:
I agree with Oleg at this point. Some loggin so the user can detected easily